### PR TITLE
Add BEM to project

### PR DESCRIPTION
**Pull request by marcbendixen**
Refactoring styles and classes with BEM

### CODE REVIEW

File: css/components/text-block.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.text-block {
```
Comment (macfrei) [inline css/components/text-block.css:1]: This could be a utility class.

File: css/components/two-columns-list.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
+.two-columns-list {
```
Comment (macfrei) [inline css/components/two-columns-list.css:1]: I'd name this after the component where you use this in. Otherwise this could be a utility class.

**Review by macfrei — COMMENTED** [review]
Well done! If you have a question, please get back to me!

File: css/components/text-block.css (hunk 1 above)
Comment (marcbendixen) [reply css/components/text-block.css]: Fixed with latest commit.

File: css/components/two-columns-list.css (hunk 2 above)
Comment (marcbendixen) [reply css/components/two-columns-list.css]: Also fixed with latest commit :-)

**Pull request merged**